tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors_test.ts

```diff
@@ -31,18 +36,20 @@ import {
   createDebuggerState,
   createState,
   createTestExecutionData,
+  createTestExecutionDigest,
+  createTestInfNanAlert,
 } from '../testing';
 
 describe('debugger selectors', () => {
   describe('getAlertsLoaded', () => {
-    it('Returns correct NOT_LOADED state', () => {
+    it('returns correct NOT_LOADED state', () => {
       const state = createState(createDebuggerState());
       const alertsLoaded = getAlertsLoaded(state);
       expect(alertsLoaded.state).toBe(DataLoadState.NOT_LOADED);
       expect(alertsLoaded.lastLoadedTimeInMs).toBe(null);
     });
 
-    it('Returns correct LOADING state', () => {
+    it('returns correct LOADING state', () => {
```

Comment:
Thanks for lowering the case :) I was a bit weirded out but did not bother to say anything